src/rev_ai/apiclient.py

```diff
@@ -28,51 +28,79 @@ def __init__(self, api_key):
             dashboard on rev.ai
         """
         if not api_key:
-            raise ValueError("API Key cannot be empty.")
+            raise ValueError('api_key must be provided')
+
         self.session = requests.Session()
         self.session.headers.update({
-            'Authorization': 'Bearer {api_key}'.format(api_key=api_key),
+            'Authorization': 'Bearer {}'.format(api_key),
             'User-Agent': 'python_sdk'
         })
 
-    def submit_job_url(self, media_url, options):
+    def submit_job_url(
+            self, media_url,
+            metadata=None,
+            callback_url=None,
+            skip_diarization=False):
         """Submit media given a URL for transcription.
         The audio data is downloaded from the URL.
 
         :param media_url: web location of the media file
-        :param options: JobSubmitOptions object for the job
+        :param metadata: info to associate with the transcription job
+        :param callback_url: callback url to invoke on job completion as a
+                             webhook
+        :param skip_diarization: should rev.ai skip diaization when
+                                 transcribing this file
```

Comment:
i'm curious what is the acceptable line length for python, like if it was c# i would say the lines above should not be wrapped

Reply:
Acceptable line length is 80 characters

Reply:
just curious why 80? where is it coming from?

Reply:
I'm not sure. Maybe @JDongian can give a more indepth reason but it is the standard
https://www.python.org/dev/peps/pep-0008/

Reply:
I think we can set it to be longer if needed - I think most change it to like 100 or 120 or just plain ignore it